src/docs/release-notes/R23/FullNotes.mdx

```diff
@@ -5,12 +5,17 @@
 
 ### Angular
 
--   Updated packages to support Angular Material v13
--   Dropped support for Ionic
+-   Updated packages to support Angular Material v13.
+-   Added `blui-toolbar-menu` component.
+-   Dropped support for Ionic.
 
 ### React
 
--   Updated packages to support MUI v5
+-   Updated packages to support MUI v5.
+-   Updated theme structures to support MUI v5.
```

Comment:
Redundancy: take this line out please.

Reply:
I see
-   Updated packages to support MUI v5.
-   Updated theme structures to support MUI v5.
-   Updated packages to support React v18.
-   Added CommonOverrides documentation and examples.
-   Updated Table styles.